copernicusmarine/python_interface/open_dataset.py

```diff
@@ -59,50 +59,51 @@ def open_dataset(
     credentials_file: Optional[Union[pathlib.Path, str]] = None,
 ) -> xarray.Dataset:
     """
-    Load an xarray dataset using "lazy-loading" mode from a Copernicus Marine data source using either the ARCO series protocol.
+    Load an xarray dataset using 'lazy-loading' mode from a Copernicus Marine data source using either the ARCO series protocol.
```

Comment:
```suggestion
    Load an xarray dataset using 'lazy-loading' mode from a Copernicus Marine data source using either of the ARCO series protocol.
```
Right? Not sure sounds weird

Reply:
easier and more to the point

Reply:
I like that it says that there are the two sources though, I don't see why getting rid of it! My comment was only about the english formulation.

But yeah both works